src/state.test.ts

```diff
@@ -29,7 +29,13 @@ suite('State Test Suite', () => {
           };
         addAutoCompletionEvent(event);
         const result = getAutoCompletionEventQueue();
-        assert.deepStrictEqual(result, [event]);
+
+        const normalizeTimestamp = (event: Partial<AutoCompletionEvent> & { timestamp: Date | string }): AutoCompletionEvent => ({
```

Comment:
Not sure how this test was passing before. I get this failure from `main` locally:

![Screenshot 2025-03-13 at 10 22 12 PM](https://github.com/user-attachments/assets/4d7a1b9b-65b2-461b-9b43-fe55c5461528)

Note that the timestamps match, it's just that the one from `getAutoCompletionEventQueue` is not a `Date`